workspaces/src/network/account.rs

```diff
@@ -17,6 +20,44 @@ impl Account {
     pub(crate) fn signer(&self) -> &InMemorySigner {
         &self.signer
     }
+
+    /// Call a contract on the network specified within `worker`, and return
+    /// a Builder object that we will make use to populate the rest of the
+    /// call details.
+    pub fn call<'a, T: Network>(
+        &self,
+        worker: &'a Worker<T>,
```

Comment:
I wonder if the `Account` should just store an `Arc<Worker>` internally, so that the user doesn't have to pass it everywhere?

Reply:
That sounds like a good idea to me

Reply:
Won't be able to have this in this PR unfortunately since we'd require `Account` to take a type parameter. Let's just wait until we resolve https://github.com/near/workspaces-rs/issues/31 and purely just get the `Worker` type without the type parameter

Reply:
Added an issue for this: https://github.com/near/workspaces-rs/issues/37